rust/processor/README.md

```diff
@@ -25,8 +25,10 @@ Indexer GRPC parser is to indexer data processor that leverages the indexer grpc
         indexer_grpc_http2_ping_timeout_in_secs: 10
         number_concurrent_processing_tasks: 10
         auth_token: AUTH_TOKEN
-        starting_version: 0 # optional
```

Comment:
nit: update the document

Reply:
Done

Reply:
sorry for being late on this.

a question about this: if `starting_version` is specific, does it also mean that processing start from this version regardless of db version?

Reply:
I've added a new field called "override_starting_version." The logic is straightforward:
1) If "override_starting_version" exists and "starting_version" is configured, use it regardless of the database version.
2) If "override_starting_version" doesn't exist, use the database version.